agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/agogosml/Dockerfile.agogosml

```diff
@@ -24,7 +24,7 @@ FROM base-deps as agogosml
 
 ENV PYTHONUNBUFFERED=1
 
-RUN pip install agogosml
+RUN pip install agogosml~={{cookiecutter.AGOGOSML_CLI_VERSION_MAJOR}}.{{cookiecutter.AGOGOSML_CLI_VERSION_MINOR}}
```

Comment:
Should we pin to just major or both major.minor?

Reply:
I'd use `==` to pin the exact version.

Reply:
A lot of projects don't always use semver, so it's good to defend against that.

Reply:
semver breaks when things like `0.10.0`  goes to `0.20.0` and the vendor has significant breaking changes. as clemens indicates, semver isn't followed properly.

Reply:
Okay so I will pin the exact major.minor version.